src/gsy_e/models/area/__init__.py

```diff
@@ -252,6 +252,19 @@ def get_results_dict(self):
         return {}
 
 
+class CoefficientAreaException(Exception):
+    """Exception that is raised when serializing an area."""
+
+
+def assert_coefficient_area_settings(
+        children: Optional[List], setting: Optional[float], setting_name: str) -> float:
```

Comment:
I'm not very convinced by the fact that here we had to add `setting_name` just to report it as part of the exception message. This is easily prone to mistakes (e.g. if one changes the `setting` parameter without updating the `setting_name`). Do you think there can be some other way around it?

Reply:
Hmm. I really do not have an idea how to solve this. IMO this is a case of YAGNI, or rather: If it happens, it happens, we will deal with this then. But I rather return the name of the misconfigured setting. Do you agree?